src/virt.rs

```diff
@@ -44,10 +44,8 @@ pub struct VirtContext {
     pub(crate) mode: Mode,
     /// Number of virtual PMPs
     pub(crate) nb_pmp: usize,
-    /// RISC-V Hypervisor extension available
-    pub(crate) has_h_mode: bool,
-    /// RISC-V Supervisor extension available
-    pub(crate) has_s_mode: bool,
+    /// Availables RISC-V extensions
+    pub(crate) extensions: ExtensionsCapability,
```

Comment:
Do we need to have it both in `mctx` and the `VirtContext`? I don't remember... If that's the case fine :)

Reply:
This make it easier :-)